dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@
 set(MODEL_FILES
   package.mo
   package.order
+  FallingEdgeDetection.mo
```

Comment:
I am not always sure if each block should be in Continuous or in NonLinear

Reply:
Yes, let's discuss that.